fairscale/optim/adascale.py

```diff
@@ -136,6 +136,10 @@ class AdaScale(Optimizer):
             for smoothing and mu and sigma variables. False will
             use the method in the paper's Appendix B.3.
             Default: True, which is what have been validated so far.
+        is_scaled_loss (bool):
```

Comment:
Can you add to line 132 above and refer to this option so that people know there are 2 ways to do grad_accum?